### PR TITLE
Style body and h1

### PR DESCRIPTION
**Pull request by daniil**
- Change the bg color of body
- Change the color of h1

### CODE REVIEW

**Comment by bstn-feb-2023** [comment]
Consider adding styles to h2 as well

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: lightskyblue;
+}
+
+h1 {
+  color: greenyellow;
```
Comment (bstn-feb-2023) [inline styles.css:6]: Consider using a more complementary colour than greenyellow

**Comment by daniil** [comment]
All changes have been implemented

**Comment by bstn-feb-2023** [comment]
Good to go. Ship it!

**Pull request merged**